controllers/charts.ts

```diff
@@ -5,17 +5,40 @@ const asyncHandler = require("express-async-handler");
 
 // Top 40
 const get_top_forty = asyncHandler(async (req, res, next) => {
-  const request = {
-    limit: req.query.limit ? req.query.limit : 41,
+  const { limit, phase }: { limit: number; phase: string } = {
+    limit: req.query.limit ? parseInt(req.query.limit) : 50,
+    phase: req.query.phase ? req.query.phase : "week", // week by default
   };
 
-  const tracks = await prisma.trackInfo.findMany({
-    where: { msatTotal30Days: { gt: 0 } },
-    orderBy: { msatTotal30Days: "desc" },
-    take: request.limit,
-  });
+  phase === "day" || phase === "week" || phase === "month"
+    ? null
+    : res.json({
+        success: false,
+        error: "Invalid phase, must be one of: day, week, month",
+      });
```

Comment:
nit - i think this would be better written with an if statement
```suggestion
  const validPhases = ["day", "week", "month"];
  if (!validPhases.includes(phase)) {
    res.json({
      success: false,
      error: "Invalid phase, must be one of: day, week, month",
    });
    return
  }
```
```suggestion
  phase === "day" || phase === "week" || phase === "month"
    ? null
    : res.json({
        success: false,
        error: "Invalid phase, must be one of: day, week, month",
      });
```